docs/intro/tutorial01.txt

```diff
@@ -19,6 +19,16 @@ tell Django is installed and which version by running the following command:
 
     $ python -m django --version
 
+.. note::
+
+    The ``$`` before the command is special. It indicates that you are supposed
```

Comment:
I wouldn't say it's special, I'd say it's a convention that indicates the line contains a command to be typed in, rather than output. Perhaps this could be clarified by including the output in this first command?